mapproxy/request/wms/__init__.py

```diff
@@ -185,11 +185,31 @@ class WMSMapRequest(WMSRequest):
     #pylint: disable-msg=E1102
     xml_exception_handler = None
     prevent_image_exception = False
+    dimension_params = ['time', 'elevation']
+    dimension_prefix = 'dim_'
```

Comment:
Why do we need this?

Reply:
(this is answered by the comment above, short version: spec)